Documentation/Books/Manual/SUMMARY.md

```diff
@@ -332,6 +332,7 @@
   * [User Management](Administration/ManagingUsers/README.md)
     * [In Arangosh](Administration/ManagingUsers/InArangosh.md)
 #    * [Durability](Administration/Configuration/Durability.md)
```

Comment:
@Simran-B please can we remove this entry (and open a planning ticket if needed) so that summary.md can be read without wrong format from github - thanks

Reply:
We already have a ticket: https://github.com/arangodb/planning/issues/2736
I removed the line.

Reply:
thanks